### PR TITLE
Log real events in hotels package

### PR DESCRIPTION
**Pull request by mrtnzlml**
Note - we do not have information about `hasActiveBooking`.

See: https://confluence.kiwi.com/display/MOB/Semi+native+hotels#Seminativehotels-Logging

---
Please check this before merging (do not delete this checklist):

- [ ] it works in landscape and portrait mode
- [ ] it uses `cacheConfig.force=true` if offline access doesn't make sense

### CODE REVIEW

**Comment by tbergq** [comment]
if the information about has active booking exists in the native code, it can be passed as a prop to the react native application from native code through: `initialProperties`
https://facebook.github.io/react-native/docs/integration-with-existing-apps.html

**Comment by mrtnzlml** [comment]
@tbergq I decided not to log it now until we know where to get this information. It basically means I want to be sure about this in order to implement it correctly (not blindly):

> if the information about has active booking exists in the native code

File: app/core/config/jestSetup.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+// @flow
+
+beforeEach(() => {
+  /**
+   * Jest will by swallow all `console.log` methods used in our code
```
Comment (mrtnzlml) [inline app/core/config/jestSetup.js]: Fuck, edit typo. Should be `Jest will swallow...`.

**Comment by tbergq** [comment]
Makes sense :)

**Pull request merged**